### PR TITLE
Add radians option to `angle_between_vectors` and add value ranges to bond distribution functions.

### PR DESCRIPTION
**Pull request by chrisjonesBSU**
This PR does a couple of things:

Adds a parameter so that `geometry.angle_between_vectors` can return the angle in degrees or radians. It defaults to degrees

Adds a radians option to the bond and angle distribution functions.

Adds min and max options to the distribution functions, so that it can work similar to the `gsd_rdf` function

### CODE REVIEW

File: cmeutils/geometry.py [hunk 1]
```diff
@@ -73,15 +75,22 @@ def angle_between_vectors(u, v, min_angle=True):
...
-    angle = np.rad2deg(
-        np.arccos(u.dot(v) / (np.linalg.norm(u) * np.linalg.norm(v)))
-    )
-    if angle > 90 and min_angle:
-        return 180 - angle
-    return angle
+    # Angle in radians
+    angle = np.arccos(u.dot(v) / (np.linalg.norm(u) * np.linalg.norm(v)))
+    if angle > math.pi/2 and min_angle:
+        angle = math.pi - angle
```
Comment (jennyfothergill) [inline cmeutils/geometry.py:91]: could use np.pi if you want to use one less import

**Review by jennyfothergill — COMMENTED** [review]
Might want to add tests for `theta_min`/`theta_max` and `l_min`/`l_max` just to validate that these values don't break anything. Also what happens if the min and max are set such that they are outside the distribution?

**Comment by chrisjonesBSU** [comment]
> 

Yeah, I can add a test. To your question, that's kind of the point of having the min and max values. For example, say some bond distribution only has values between 1 and 2, but you want the distribution to go from zero to 3, so you'd set min=0, max=3 and the distribution would include the entire range from min to max, even if a good portion of it is zero.

**Comment by jennyfothergill** [comment]
> > Might want to add tests for `theta_min`/`theta_max` and `l_min`/`l_max` just to validate that these values don't break anything. Also what happens if the min and max are set such that they are outside the distribution?
> 
> Yeah, I can add a test. To your question, that's kind of the point of having the min and max values. For example, say some bond distribution only has values between 1 and 2, but you want the distribution to go from zero to 3, so you'd set min=0, max=3 and the distribution would include the entire range from min to max, even if a good portion of it is zero. Unless I'm not understanding what you mean by outside the distribution.

What I mean using your example of a bond distribution with values between 1 and 2, what if you set the min and max to 1.5 to 3? Some of the distribution isn't captured. Is there anything we should do to handle that? (I don't mean these as leading questions, I'm just thinking about possible hangups.)

**Comment by chrisjonesBSU** [comment]
If that's the case then I'm pretty sure numpy.histogram just trims the distribution based on the range you give it. We could maybe add a warning if any part of the array of values returned (bond lengths, angles) lies outside of either min or max values, but it wouldn't error out.

**Pull request merged**